### PR TITLE
Update installation docs for CUDA 12.

### LINKED ISSUE #3909: [DOC]: Update cuGraph website for cuda 12
### How would you describe the priority of this documentation request

High

### Please provide a link or source to the relevant docs

https://docs.rapids.ai/api/cugraph/nightly/installation/getting_cugraph/

### Describe the problems in the documentation

All installation docs refer only to cuda 11.

- [ ] Add references to cuda 12
- [ ] Correct cuda 11 references with the valid versions
- [ ] check for required changes in docker container

### (Optional) Propose a correction

_No response_

### Code of Conduct

- [X] I agree to follow cuGraph's Code of Conduct
- [X] I have searched the [open documentation issues](https://github.com/rapidsai/cugraph/issues?q=is%3Aopen+is%3Aissue+label%3Adoc) and have found no duplicates for this bug report

### PR DESCRIPTION
**Pull request by bdice**
Closes #3909.

### CODE REVIEW

**Comment by bdice** [comment]
@acostadon Can you review and ensure that this covers the changes you wanted in #3909? I noticed this was out of date while updating some RAPIDS CI workflows.

**Review by BradReesWork — APPROVED** [review]

File: docs/cugraph/source/installation/getting_cugraph.md [hunk 1]
```diff
@@ -31,28 +31,38 @@ cuGraph Conda packages
...
 
 ```bash
-conda install -c rapidsai -c conda-forge -c nvidia cugraph cudatoolkit=11.8
+conda install -c rapidsai -c conda-forge -c nvidia cugraph cuda-version=12.0
 ```
 
-Note: This conda installation only applies to Linux and Python versions 3.8/3.10.
+Alternatively, use `cuda-version=11.8` for packages supporting CUDA 11.
+
+Note: This conda installation only applies to Linux and Python versions 3.9/3.10.
```
Comment (jakirkham) [inline docs/cugraph/source/installation/getting_cugraph.md:48]:
```suggestion
Note: This conda installation only applies to Linux and Python versions 3.9/3.10/3.11.
```

**Review by acostadon — COMMENTED** [review]
Looks good to me

File: docs/cugraph/source/installation/getting_cugraph.md (hunk 1 above)
Comment (jakirkham) [reply docs/cugraph/source/installation/getting_cugraph.md:48]: Submitted in PR: https://github.com/rapidsai/cugraph/pull/4213

**Pull request merged**